flatpak/com.matthiasnehlsen.lotti.local.yml

```
@@ -0,0 +1,71 @@
+      # Copy entire Flutter bundle to /app (Flutter expects this structure)
+      - cp -r . /app/
+      
+      # Remove 1024x1024 icons to comply with Flatpak's maximum icon size limit of 512x512
+      - find /app -name "*1024*" -type f -delete
+      
+      # Ensure executable permissions
+      - chmod +x /app/lotti
+      
+    sources:
+      - type: dir
+        path: flutter-bundle
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)